crates/ruma-state-res/Cargo.toml

```diff
@@ -15,7 +15,11 @@ edition = "2018"
 all-features = true
 
 [features]
-unstable-pre-spec = ["ruma-events/unstable-pre-spec"]
+unstable-pre-spec = ["ruma-common/unstable-pre-spec", "ruma-events/unstable-pre-spec"]
+unstable-exhaustive-types = [
+    "ruma-common/unstable-exhaustive-types",
+    "ruma-events/unstable-exhaustive-types",
+]
```

Comment:
The exhaustive types feature should not activate the same feature in other crates (except in `ruma`).

Reply:
Ugh this must have gotten lost in the shuffle, I thought I added a commit fixing this and a few others. I wasn't just ignoring your reviews :laughing: